packages/apollo/src/utils/lists.ts

```diff
@@ -108,6 +179,10 @@ export class ListHelper {
             return this._readPage
         }
 
+        if (clientPagination === 'showAll') {
```

Comment:
Remove `undefined` from return-type and return _readAll as last option (without if)